docker/Dockerfile.py

```diff
@@ -0,0 +1,37 @@
+import argparse
+import textwrap
+
+parser = argparse.ArgumentParser()
+parser.add_argument('--docker-version', required=True)
+
+args = parser.parse_args()
+
+
+dependencies = ' '.join(('git',
+                         'python-dev',
+                         'python-pip',
+                         'wget',
+                         'curl',
+                         'apt-transport-https',
+                         'ca-certificates'))
+docker_version = args.docker_version
+print textwrap.dedent("""
+    FROM ubuntu:14.04
+
+    RUN apt-get update \
+        && apt-get install -y {dependencies}
+    RUN wget https://get.docker.com/builds/Linux/x86_64/docker-{docker_version}.tgz \
+            -O /usr/local/bin/docker \
+        && chmod u+x /usr/local/bin/docker
+    RUN pip install setuptools --upgrade
+    RUN pip install toil==3.5.0a1.dev274
+
+    COPY wrapper.py /opt/pipeline/
+    COPY toil-rnaseq* /opt/pipeline/sdist.tar.gz
```

Comment:
Is this supposed to copy the sdist? If so, I would parameterize the `Dockerfile` with the exact name of the `sdist` and use that here. Toil does it that way. The wildcard could bite you if someone switched a working copy between two different versions and didn't clean in between. In that case there will be two different tarballs, both matching the wildcard.